### PR TITLE
[Merged by Bors] - chore: delay imports of Rat/Field

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Module/LinearMap/Rat.lean [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+/-!
+# Reinterpret an additive homomorphism as a `ℚ`-linear map.
+-/
+
+open Function
+
+universe u u' v w x y z
+
+variable {R R₁ R₂ R₃ k S S₃ T M M₁ M₂ M₃ N₁ N₂ N₃ ι : Type*}
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Module/LinearMap/Rat.lean:18]:
```suggestion
variable {M M₂ : Type*}
```

**Review by Ruben-VandeVelde — APPROVED** [review]
Seems reasonable once you shake

**Review by grunweg — COMMENTED** [review]
maintainer merge

**Pull request closed without merging**